src/ducks/apps/components/ApplicationDetails.jsx

```diff
@@ -24,23 +25,37 @@ export const ApplicationDetails = ({t, lang, app: { description, icon, installed
       <div className='sto-app-content'>
         <h2>{editor ? `${editor} ${appName}` : appName}</h2>
         <p>{appDesc}</p>
-        <button
-          role='button'
-          onClick={() => openApp(related)}
-          className='coz-btn coz-btn--regular'
-        >
-          <Icon
-            icon='openwith'
-            width='10px'
-            height='10px'
-          /> {t('app.open')}
-        </button>
-        <Link
-          to={`/${parent}/${slug}/manage`}
-          className='coz-btn coz-btn--danger-outline'
-        >
-          {t('app.uninstall')}
-        </Link>
+        {
+          installed
+          ? <div>
+            <button
+              role='button'
+              onClick={() => openApp(related)}
+              className='coz-btn coz-btn--regular'
+            >
+              <Icon
+                icon='openwith'
+                width='10px'
+                height='10px'
+              /> {t('app.open')}
+            </button>
+            <Link
+              to={`/${parent}/${slug}/manage`}
+              className='coz-btn coz-btn--danger-outline'
+            >
+              {t('app.uninstall')}
+            </Link>
+          </div>
+          : <Link
+            to={`/${parent}/${slug}/manage`}
+            className='coz-btn coz-btn--regular'
+          >
+            <svg className='sto-app-icon--button'>
```

Comment:
According to the [`Icon` component](https://github.com/cozy/cozy-ui/tree/master/react/Icon) from cozy-ui, you could use it with a custom icon.
```js
<Icon icon={cozySmileWhiteIcone} />
```
Also there is a typo s/cozySmileWhiteIcome/cozySmileWhiteIcon

Reply:
:+1: